rebar.config

```diff
@@ -20,7 +20,7 @@
 	{erlando, {git, "https://github.com/travelping/erlando.git", {tag, "1.0.3"}}},
 	{netdata, {git, "https://github.com/RoadRunnr/erl_netdata.git", {ref, "cbd6eaf"}}},
 	{gtplib, "2.0.1"},
-	{pfcplib, "2.0.0"},
+	{pfcplib, {git, "https://github.com/travelping/pfcplib.git", {branch, "feature/bbf-tr-459"}}},
```

Comment:
Do we need use `feature/bbf-tr-459"` branch or maybe we can merge and release `pfcplib`?

Reply:
Can be used now `{pfcplib, "2.0.1"}`.